pom.xml

```diff
@@ -47,7 +47,14 @@
                 de.remsfal.service.entity.dto\**
         </sonar.cpd.exclusions>
     </properties>
-
+    <dependencies>
```

Comment:
The dependency is only in the service module required. Therefore, please move this dependency to the remsfal-service/pom.xml